addon/models/relationship.js

```diff
@@ -0,0 +1,7 @@
+import Model from 'ember-data/model';
+
+export default Model.extend({
+    target_type: DS.attr('string'),
```

Comment:
In general, try to follow JS conventions for names where possible (`targetType` vs `target_type`). Eg, the application adapter should take care of converting names from camelcase to snakecase and back, in cases where the API response does not match ember data conventions.

I wish I could say I was being pedantic, but Ember has a *lot* of magical assumptions based on how fields are named.